core/Makefile

```diff
@@ -26,8 +26,8 @@ test: $(MAKEDIRS)
 
 first:
 	$(MAKE) -C kazoo/ $(ACTION)
+	$(MAKE) -C kazoo_amqp/ $(ACTION)
 	$(MAKE) -C kazoo_data/ $(ACTION)
```

Comment:
Now that you're removing the kazoo transform, is the `first` target really needed?